compiler/testData/checkLocalVariablesTable/underscoreNames.kt

```diff
@@ -23,7 +23,6 @@ fun box() {
 // METHOD : UnderscoreNamesKt$box$1.invoke(LA;Ljava/lang/String;I)Ljava/lang/String;
 // VARIABLE : NAME=x TYPE=D INDEX=4
 // VARIABLE : NAME=y TYPE=C INDEX=6
-// VARIABLE : NAME=q TYPE=Ljava/lang/String; INDEX=16
```

Comment:
The change here is that in fact there is no place where the q is visible in the debugger before and after this change. 

The code is:

```for ((_, q) in arrayOfA) {

}```

The q is in the local variable table covering exactly one `iinc` instruction in the generated code before this change. That `iinc` instruction is part of the condition and therefore we do not put in the `q` at all. From a debugger perspective it doesn't matter.

Reply:
I suppose the point of the test is not to check behavior for an empty loop, but to check that a named destructured variable is displayed correctly after `_`? Maybe add some no-op instruction like `Unit` to the loop body to continue checking that `q` is still there for the normal case (since loops are usually not empty...)?

Reply:
Done.